README.md

```diff
@@ -1,33 +1,80 @@
+**Delivery mindset**
+
+We believe in being pragmatic and moving fast, spending our energy on the right things, shipping code and then rapidly iterating on it. 
+
+_Bbehaviours_:
+- We simplify our work, focusing on getting it to the user as early as possible.
+- We practice [continuous delivery](https://en.wikipedia.org/wiki/Continuous_delivery)
+- We are trying to [make the change easy, before the easy change](https://twitter.com/KentBeck/status/250733358307500032).   
+- We aim to solve issues rapidily priorising them based on impact they may have.
+
+**Inclusiveness & Respect** 
 
 We believe that our culture is important, and we all contribute to it regardless of our role or level.
 
->Every one of us has multiple interactions with a variety of people across the department and company every day. Each one of those is an opportunity to express and refine our culture. This written statement of our culture is shared, and anyone can [contribute](CONTRIBUTING.md).
+_Behaviours_:
+ - We ensure to treat everyone we encounter with respect independently of their skillsets and levels of ability
+ - Every one of us has multiple interactions with a variety of people across the department and company every day. Each one of those is an opportunity to express and refine our culture. This written statement of our culture is shared, and anyone can [contribute](CONTRIBUTING.md).
+ - We respect the opinion of others and are gracious in our interactions, whether meeting physically or digitally, whether friend or stranger, whether employee, customer or commenter.
+ - We come from different backgrounds, work in different roles, are different ages, and therefore have different levels of experience. All in all, we pull each other up, not bring each other down.
 
 
-We believe that all of us want to - and can - make a difference to the Guardian.
+**Working as a team**
+
+We believe that the team is everything and our lowest common unit, set up with the necessary skills to succeed.
 
->We choose to work here because we believe in what the Guardian stands for. We're excited at contributing to the Guardian's future. 
+_Behaviours_:
+ - We fail and succeed together as a team.
+ - We are independent, accountable and share responsibility. We conduct ourselves with humility, kindness and a calm manner. We debate ideas, not people and we share constructive feedback openly. 
+ - We take the time needed to listen to and help our colleagues with any problem they might have from anywhere within the business.
+ - We value the experience of people in the team and the different constraints within their lives. 
+ - We celebrate success and positive behaviour, recognizing everyone’s contribution while treating every moment including tasks and failures as a learning opportunity.
 
-We believe in treating everyone we encounter with respect.
+**We are data driven** 
```

Comment:
https://segment.com/resources/data-strategy/data-driven-vs-data-informed/

> Data-driven means that you always rely on data to make your decisions. If you’re trying to improve the UX of your product, you’ll run tests to get data to find the best approach.

> Data-informed means that you use data alongside your unique experience, user research, and other inputs to make decisions. With a data-informed approach, data is just one part of your decision-making process.

I believe that we're much more aligned to the Data Informed approach - we use the data to help make decisions alongside the historic, editorial, user experience and performance contexts.

Reply:
@gtrufitt Thanks Gareth, great point! I have now amended to use `Data informed`